pilot/pkg/security/authn/v1beta1/policy_applier.go

```diff
@@ -134,6 +134,14 @@ func (a *v1beta1PolicyApplier) AuthNFilter(proxyType model.NodeType) *http_conn.
 	}
 }
 
+func (a *v1beta1PolicyApplier) AcceptMtls(port uint32) bool {
+	if a.consolidatedPeerPolicy == nil {
```

Comment:
This is redundant with `getMutualTLSModeForPort`

Reply:
good point. could just return that func directly.

Reply:
done